src/devices/Button/ButtonBase.cs

```diff
@@ -78,17 +80,24 @@ public ButtonBase()
         /// </summary>
         /// <param name="doublePress">Max ticks between button presses to count as doublepress.</param>
         /// <param name="holding">Min ms a button is pressed to count as holding.</param>
-        public ButtonBase(TimeSpan doublePress, TimeSpan holding)
+        /// <param name="debounceTime">The amount of time during which the transitions are ignored, or zero</param>
+        public ButtonBase(TimeSpan doublePress, TimeSpan holding, TimeSpan debounceTime)
```

Comment:
new argument should be optional

Reply:
Right, I'll fix it now. Anyway the button is not yet in a released nuget package (unless it has just been released this night :-D)